apps/emqx_authz/etc/emqx_authz.conf

```diff
@@ -1,4 +1,18 @@
 emqx_authz:{
+    ## Enable ACL check.
+    ##
+    ## @doc emqx_authz.enable
+    ## ValueType: Boolean
+    ## Default: false
+    enable: true
+
+    ## The action when authorize check reject current operation
+    ##
+    ## @doc emqx_authz.deny_action
+    ## ValueType: reply | disconnect
+    ## Default: reply
+    deny_action: reply
```

Comment:
1. I think the deny_action should be under the scope of zone.external.mqtt? Why would it be inside authz?

2. Even then, `reply` is an unclear concept in authz?